api/src/main/java/org/openmrs/module/dhisreport/api/db/hibernate/HibernateDHIS2ReportingDAO.java

```diff
@@ -180,8 +180,8 @@ public String evaluateDataValueTemplate( DataValueTemplate dvt, Period period, L
         {
             query.setParameter( "locationId", location.getId().toString() );
         }
-        query.setParameter( "startOfPeriod", period.getStart() );
-        query.setParameter( "endOfPeriod", period.getEnd() );
+        query.setParameter( "startOfPeriod", period.getStartDate() );
+        query.setParameter( "endOfPeriod", period.getEndDate() );
```

Comment:
@maurya here are changes for getEndDate for HibernateDHIS2ReportingDAO.java file